api.md

```diff
@@ -1007,7 +1007,20 @@ AS
         FROM conditions
         GROUP BY time_bucket('1day', timec), location, humidity, temperature;
 ```
+
+Create an up-to-date continuous aggregate view that runs whenever new data is received.
+```sql
+CREATE VIEW continuous_aggregate_view( timec, minl, sumt, sumh )
+WITH ( timescaledb.continuous,
+    timescaledb.refresh_lag = '5 hours',
+    timescaledb.refresh_interval = '-1h' )
+AS
+    SELECT time_bucket('1day', timec), min(location), sum(temperature), sum(humidity)
+        FROM conditions
+        GROUP BY time_bucket('1day', timec), location, humidity, temperature;
+```
```

Comment:
Add a warning about the performance implications for negative `refresh_lag`?

Reply:
Done!

Reply:
Oh, I mean't an actual `>:WARNING:` type tip.

Reply:
Ok, @gayyappan actually suggested just creating a tip for this, so I added the warning text into the large tip